test_regress/t/t_vpi_module.cpp

```diff
@@ -48,24 +48,55 @@ unsigned int main_time = 0;
         return __LINE__; \
     }
 
+#define CHECK_RESULT_Z(got) \
+    if (got) { \
+        printf("%%Error: %s:%d: GOT = !NULL  EXP = NULL\n", FILENM, __LINE__); \
+        return __LINE__; \
+    }
+
 #define CHECK_RESULT_CSTR(got, exp) \
     if (strcmp((got), (exp))) { \
         printf("%%Error: %s:%d: GOT = '%s'   EXP = '%s'\n", FILENM, __LINE__, \
                (got) ? (got) : "<null>", (exp) ? (exp) : "<null>"); \
         return __LINE__; \
     }
 
+void modDump(vpiHandle it, int n) {
+    while (vpiHandle hndl = vpi_scan(it)) {
+        char* nm = vpi_get_str(vpiName, hndl);
+        for (int i = 0; i < n; i++)
+            printf("    ");
```

Comment:
Run clang-format on this file please.

Reply:
The best I've got on this particular machine is `clang-format-9`. I got the warning about not using 10.0.  Hopefully that's fine.